newIndex.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=
+    , initial-scale=1.0">
+
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="newStyles.css">
+    <link href="https://fonts.cdnfonts.com/css/new-york-extra-large" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/sf-mono" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/sf-pro-display" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/sf-pro-display" rel="stylesheet">
+    <title>Nordic Rose</title>
+</head>
+
+<body>
+    <artcile>
```

Comment:
no need to wrap your header into `article` tag. read [here](https://developer.mozilla.org/ru/docs/Web/HTML/Element/article) about `article` tag semantics